doc/source/install/backend-windows-install.rst

```diff
@@ -37,12 +37,12 @@ Prérequis
     - `PowerShell 3.0+ <http://www.microsoft.com/fr-fr/download/details.aspx?id=40855>`_. Uniquement pour les PC tournant sous Windows 7 ou antérieur (installé par défaut avec Windows depuis).
     - `Git <http://git-scm.com/download/win>`_ (Git pour Eclipse ne suffit pas ; associez les .sh).
     - `gettext <https://www.gnu.org/software/gettext/>`_.
-- `Téléchargez et installez Python 2.7 <https://www.python.org/download/releases/2.7/>`_.
+- `Téléchargez et installez Python 3.4 <https://www.python.org/downloads/release/python-344/>`_.
```

Comment:
La version 3.4.6 est sortie près d'un an après : https://www.python.org/downloads/release/python-346/

Reply:
Tiens, d’ailleurs il y a même [3.4.7](https://www.python.org/downloads/release/python-347/).

Reply:
Par contre, c’est indiqué « Also, Python 3.4.6 has only been released in source code form ».

Reply:
Du coup ouais 3.4.7 c'est mieux !

Reply:
Ouais mais visiblement il n’y a pas de binaires, il faut tout compiler sois-même, non ? Et il faut peut-être aussi faire attention à la version de Python sur la production et sur la CI.

(Après je ne connais pas énormément Python donc bon)

Reply:
OK je pensais que 3.4.7 avait des binaires mais pas 3.4.6. Du coup ouais, https://www.python.org/downloads/release/python-344/. Mais ce serait bien de passer à une version plus récente.